specification/loadtestservice/resource-manager/Microsoft.LoadTestService/preview/2021-12-01-preview/examples/LoadTests_Delete.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parameters": {
+    "api-version": "2021-12-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "dummyrg",
+    "loadTestName": "myLoadTest"
+  },
+  "responses": {
+    "200": {},
+    "202": {},
```

Comment:
>202 [](http://example.com/codeflow?start=5&length=3)

do you accept 202 for delete ?

Reply:
It's an LRO which can send 202 Accespted as API response.